src/Hpack/Config.hs

```diff
@@ -231,17 +231,19 @@ data ExecutableSection = ExecutableSection {
   executableSectionMain :: Maybe FilePath
 , executableSectionOtherModules :: Maybe (List Module)
 , executableSectionGeneratedOtherModules :: Maybe (List Module)
+, executableSectionRtsOptions :: Maybe (List String)
```

Comment:
@tfausak this specifies the syntax, so yes, it can be a list, a string, or it can be absent.